### DISCUSSION TITLE
Recent tab

### DISCUSSION THREAD

**Discussion by manojvignesh** [zen-browser/desktop#1099] [Q&A] [answered]
Is there a shortcut to recent tab?

Like using `ctrl + tab` to switch to previously viewed tab instead of next tab

**Comment by giuli635** [comment]
No, there is not, but you can enable a setting to make Ctrl+Tab to work like that

**Comment by DaDecky** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/c604c5be-43ab-44ff-852b-56a649c58506)

Check the "Ctrl+Tab cycles through tabs in recently used order"
